app/util/data_preparation/api/confluence_clients.py

```diff
@@ -139,6 +139,11 @@ def get_total_pages_count(self):
         response = self.get(api_url, 'Could not get issues count')
         return response.json().get('totalSize', 0)
 
+    def get_collaborative_editing_status(self):
+        api_url = f'{self.host}/rest/synchrony-interop/status'
+        response = self.get(api_url, error_msg='Could not get collaborative editing status').json()
```

Comment:
It's better not to mix response and response serialization.
```
response = self.get(api_url, error_msg='Could not get collaborative editing status')
return response.json
```

Reply:
Done